FrontEnd/src/components/RestorePassword/components/restorepassword/restorepassword-form/RestorePasswordFormContent.js

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import EyeInvisible from '../../../../authorization/EyeInvisible';
+import EyeVisible from '../../../../authorization/EyeVisible';
+import styles from './RestorePasswordFormContent.module.css';
+
+export function RestorePasswordFormContentComponent({ setIsValid }) {
+  const navigate = useNavigate();
+  const { uid, token } = useParams();
+  const [showPassword, setShowPassword] = useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
+
+  const togglePassword = () => {
+    setShowPassword(!showPassword);
+  };
+
+  const toggleConfirmPassword = () => {
+    setShowConfirmPassword(!showConfirmPassword);
+  };
+
+  const errorMessageTemplates = {
+    required: 'Обов’язкове поле',
+    password: 'Пароль не відповідає вимогам',
+    confirmPassword: 'Паролі не збігаються',
+  };
+
+  const passwordPattern = /^(?=.*[0-9])(?=.*[a-zA-Z])[a-zA-Z0-9]{8,20}$/;
```

Comment:
I think we want to use the same patter for Register and Login. It makes sense to export it from one place

Reply:
Fixed